rfc/github-org-migration.md

```diff
@@ -0,0 +1,425 @@
+Once a repository is successfully migrated, it will be marked as [x].
+
+The following repos lists were computed with: `curl -Ls -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $GITHUB_TOKEN" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/orgs/$ORG/repos\?per_page\=100\&page\=$PAGE | jq -r '.[] | "- [ ] [" + .name + "](" + .html_url + ")"' |`, where `$ORG ∈ {"web3-storage", "nftstorage", "filecoin-saturn"}`, `GITHUB_TOKEN` is a token with access to all orgs, and `$PAGE ∈ [1,2]`. Further reference [here](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories).
+
+### [Web3 Storage](https://github.com/web3-storage)
+
+- [ ] [ipfs-car](https://github.com/web3-storage/ipfs-car)
+- [ ] [web3.storage](https://github.com/web3-storage/web3.storage)
+- [ ] [web3-file](https://github.com/web3-storage/web3-file)
+- [ ] [go-w3s-client](https://github.com/web3-storage/go-w3s-client)
+- [ ] [wrangler-action](https://github.com/web3-storage/wrangler-action)
+- [ ] [add-to-web3](https://github.com/web3-storage/add-to-web3)
+- [ ] [files-from-path](https://github.com/web3-storage/files-from-path)
+- [ ] [multipart-parser](https://github.com/web3-storage/multipart-parser)
+- [ ] [example-image-gallery](https://github.com/web3-storage/example-image-gallery)
+- [ ] [broker](https://github.com/web3-storage/broker)
+- [ ] [web3-schema](https://github.com/web3-storage/web3-schema)
+- [ ] [example-forum-dapp](https://github.com/web3-storage/example-forum-dapp)
+- [ ] [fauna-dump](https://github.com/web3-storage/fauna-dump)
+- [ ] [db-migration-pipeline](https://github.com/web3-storage/db-migration-pipeline)
+- [ ] [fauna-dumpify](https://github.com/web3-storage/fauna-dumpify)
+    - Old DB migrations for old products;
+    - It SHOULD be archived;
+    - It SHOULD NOT be migrated.
+- [ ] [admin.storage](https://github.com/web3-storage/admin.storage)
+- [ ] [hydra-booster](https://github.com/web3-storage/hydra-booster)
+- [ ] [ipns-publisher](https://github.com/web3-storage/ipns-publisher)
+- [ ] [specs](https://github.com/web3-storage/specs)
+- [ ] [consul-cluster-go-ipfs](https://github.com/web3-storage/consul-cluster-go-ipfs)
+- [ ] [repin-unpinned-cids](https://github.com/web3-storage/repin-unpinned-cids)
+- [ ] [parse-link-header](https://github.com/web3-storage/parse-link-header)
+- [ ] [js-replica](https://github.com/web3-storage/js-replica)
+- [ ] [w3up-archived](https://github.com/web3-storage/w3up-archived)
+- [ ] [upload-protocol](https://github.com/web3-storage/upload-protocol)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [ucanto](https://github.com/web3-storage/ucanto)
+- [ ] [w3up](https://github.com/web3-storage/w3up)
+- [ ] [ucanto-name-system](https://github.com/web3-storage/ucanto-name-system)
+- [ ] [alt-gateway](https://github.com/web3-storage/alt-gateway)
+- [ ] [link.web3.storage](https://github.com/web3-storage/link.web3.storage)
```

Comment:
@vasco-santos do you know what this is?

Reply:
it was some tests we did in our team week in Miami to test the service worker deployment. I think we can archive